src/libcore/raw.rs

```diff
@@ -93,7 +93,32 @@
 #[repr(C)]
 #[derive(Copy, Clone)]
 #[allow(missing_debug_implementations)]
+#[unstable(feature = "raw", issue = "27751")]
 pub struct TraitObject {
     pub data: *mut (),
     pub vtable: *mut (),
 }
+
+/// Returns the value of the discriminant for the enum variant in `v`.
```

Comment:
Wait, I don't understand. You mean if the enum is _not_ `#[repr(u64)]`?

On Tue, Jul 12, 2016 at 1:46 PM, Alex Burka <durka42@gmail.com> wrote:

> Will do.
>
> On Tue, Jul 12, 2016 at 1:43 PM, Steven Fackler <notifications@github.com>
> wrote:
>
>> In src/libcore/raw.rs
>> <https://github.com/rust-lang/rust/pull/34785#discussion_r70485187>:
>>
>> >  pub struct TraitObject {
>> >      pub data: *mut (),
>> >      pub vtable: *mut (),
>> >  }
>> > +
>> > +/// Returns the value of the discriminant for the enum variant in `v`.
>>
>> It is probably worth adding a warning that discriminants may not compare
>> in the correct order if the enum is #[repr(u64)] due to overflow.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/rust-lang/rust/pull/34785/files/609c17e8d0ceac437261df2c6b4231225f1faf08#r70485187>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/AAC3n4IaNbDuCLcSiYjaKILI3yho7yvLks5qU9IrgaJpZM4JKnnt>
>> .
>>
>
>